.github/workflows/publish.yml

```diff
@@ -27,3 +27,8 @@ jobs:
         run: node_modules/.bin/vsce publish --packagePath vscode-shopify-ruby.vsix
         env:
           VSCE_PAT: ${{ secrets.VSCE_PAT }}
+
+      - name: Publish extension on OpenVSX
+        run: |
+          yarn run package
+          node_modules/.bin/ovsx publish vscode-shopify-ruby.vsix -p ${{ secrets.OPENVSX_TOKEN }} --yarn
```

Comment:
I'm not familiar with that command, but `ovsx` is only a development dependency. What's the advantage of using `dlx`?

Reply:
Ah, I see. I think it's better to have the explicit dependency to ensure we're running the same versions of `ovsx` and its dependencies in all environments. Otherwise it could be more difficult when troubleshooting.